packages/forms-web-app/src/routes/config.js

```diff
@@ -0,0 +1,8 @@
+const routes = {
+	external: {
+		emailSignUpURL: 'https://nitestaz.planninginspectorate.gov.uk/email-subscription.php'
```

Comment:
This will fail in production as that is a test URL.

Reply:
We should use an environment variable to generate emailURL in the same way as all other documents in the library.
This should be set by the environment variable which may already be available, think the base URL will be
Dev
https://nitestaz.planninginspectorate.gov.uk
Test
https://nipreprod.planninginspectorate.gov.uk
Prod
https://infrastructure.planninginspectorate.gov.uk

Reply:
@Philip-Philippou1 Thank you for that! I will get it added to the env variables.

Reply:
@Philip-Philippou1 I have added a variable to the config under routes.external.nsipEmailSubscriptionUrl. This uses the already existing environment variable `NSIP_BASE_URL` and concatenates it with the new `NSIP_EMAIL_SUBSCRIPTION_PATH`. This will have to be added to the environment but it currently fallback to `https://infrastructure.planninginspectorate.gov.uk/email-subscription.php`.

Reply:
Hi @Philip-Philippou1, following our conversation regarding the `NSIP_BASE_URL` are you happy to keep it how it currently is, falling back to the production pins production URL `https://infrastructure.planninginspectorate.gov.uk` for now? I will raise with Chris Bungay and create a separate branch if changes are needed.